pkgs/tools/typesetting/tex/texlive/tlpdb-overrides.nix

```diff
@@ -337,7 +339,9 @@ in lib.recursiveUpdate orig rec {
 
   #### dependency changes
   # it seems to need it to transform fonts
-  xdvi.deps = (orig.xdvi.deps or []) ++  [ "metafont" ];
+  xdvi.deps = (orig.xdvi.deps or [ ]) ++ [ "metafont" ];
+
+  mltex.deps = (orig.mltex.deps or [ ]) ++ [ "pdftex" ];
```

Comment:
Forgot to ask: why is this appearing now? (I'd check myself, but I am struggling to build due to mirror issues.)

Reply:
Found the reason. It's a format built with pdftex. I guess it used to depend transitively on pdftex, but now it does not.